### DISCUSSION TITLE
Compatibility with luxtronik software

### DISCUSSION THREAD

**Discussion by aldrnikolajev** [BenPru/luxtronik#279] [Q&A]
Hey, is it safe to update heatpump  (Alpha Innotec swc122h3) to luxtronik's latest (3.90) version related to my hp?

**Comment by Mehli12** [comment]
I use 3.90 with Luxtronik2 2024.9.30. 
Working fine from my side.

**Reply by aldrnikolajev** [reply]
Thanks, will try to update shortly.
